lib/cmds/space_cmds/environment_cmds/create.js

```diff
@@ -88,35 +124,12 @@ async function environmentCreate({
 
   logging.success(
     `Successfully created environment ${environment.name} (${
-      environment.sys.id
+    environment.sys.id
     }) ${source ? `with source ${source}` : ''}`
   )
 
   if (awaitProcessing) {
-    const DELAY = 3000
-    const MAX_NUMBER_OF_TRIES = 100
-    let count = 0
-
-    logging.log('Waiting for processing...')
-
-    while (count < MAX_NUMBER_OF_TRIES) {
-      const status = (await space.getEnvironment(environment.sys.id)).sys.status
-        .sys.id
-
-      if (status === 'ready' || status === 'failed') {
-        if (status === 'ready') {
-          logging.success(
-            `Successfully processed new environment ${environment.name} (${environment.sys.id})`
-          )
-        } else {
-          logging.error('Environment creation failed')
-        }
-        break
-      }
-
-      await Promise.delay(DELAY)
-      count++
-    }
+    checkAwaitProcessing(space, environment, processingTimeout)
```

Comment:
We should log some type of timeout-related message for users after awaiting processing in the case where the environment is not ready. This way the user knows this and can bump their timeout values.

Reply:
Thank you for the feedback, I'll this change in another PR 👍